pkgs/development/libraries/science/math/osqp/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation {
+  pname = "osqp";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner  = "oxfordcontrol";
+    repo   = "osqp";
+    rev    = "0baddd36bd57ec1cace0a52c6dd9663e8f16df0a";
```

```suggestion
    rev = version;
```

also need to make derivation a recursive set: ``stdenv.mkDerivation rec { ...``

---

Also, generally there's no need to add spaces before the equals signs here to align them.